### PR TITLE
sineInterpolation

### PR DESCRIPTION
**Pull request by bettinaselig**
Added a function that maps a value between 0.0 and 1.0 to a sine-like function.

### CODE REVIEW

**Comment by bettinaselig** [comment]
@marcusnaslund Peer review?

File: source/system/math.ooc [hunk 1]
```diff
@@ -240,6 +240,15 @@ extend Float {
...
 		result
 	}
+	sineInterpolation: func -> Float {
+		result := 0.0f
+		absoluteValue := this absolute
+		if (absoluteValue > 1.0f)
+			result = 1.0f
+		else
+			result = (sin((absoluteValue - 0.5f) * pi) + 1.0f) / 2.0f
+		result
```
Comment (marcusnaslund) [inline source/system/math.ooc]: The above five lines can be written simply as
`absoluteValue > 1.f ? 1.f : (sin((absoluteValue - 0.5f) * pi) + 1.0f) / 2.0f`

Comment (bettinaselig) [inline source/system/math.ooc]: Yes, was thinking about that, but thought it would be too much in one line. But since you suggest it, I will change it :-)

**Comment by bettinaselig** [comment]
Done.

File: source/system/math.ooc [hunk 2]
```diff
@@ -240,6 +240,11 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	sineInterpolation: func -> Float {
+		result := 0.0f
```
Comment (marcusnaslund) [inline source/system/math.ooc]: You forgot to remove this. :)

Comment (bettinaselig) [inline source/system/math.ooc]: Already on it :-) Thanks!

**Comment by bettinaselig** [comment]
Done.

**Pull request merged**